dtfs-central-api/src/types/utilisation-report-status.ts

```diff
@@ -0,0 +1,46 @@
+import { ObjectId } from 'mongodb';
+import { TFMUser } from './users';
+
+export type ReportStatus = 'RECONCILIATION_COMPLETED' | 'REPORT_NOT_RECEIVED';
```

Comment:
I only added these statuses for now as they were the only ones I needed and wanted to leave it as something to add in future as and when we need the extra statuses (so that someone doesn't come and look at this PR in the future and wonder why I added more types which I didn't need at that point). I'm not sure whether or not that logic is the best practice though, so I am happy to add the rest of the statuses